README.md

```diff
@@ -1,15 +1,17 @@
 # OSV - Open Source Vulnerabilities
 
-OSV is a [vulnerability database] and triage infrastructure for open source
-projects aimed at helping both open source maintainers and consumers of open
-source.
+https://osv.dev is a [vulnerability database] and triage infrastructure for
```

Comment:
Can we linkify with osv.dev, it looks weird with https:// in so many places.

Reply:
Or we can even say just osv.dev in places

Reply:
Good poing, done!